examples/step-3/doc/results.dox

```diff
@@ -120,15 +120,12 @@ suggestions:
   (here 1e-12) of -1 and 1. Try this immediately after calling
   GridGenerator::hyper_cube(), as before:
   @code
-  for (const auto &cell : dof_handler.active_cell_iterators())
-    {
-        for (auto &face : cell->face_iterators())
-          if ((std::fabs(face->center()(1) - (-1.0)) < 1e-12) ||
-              (std::fabs(face->center()(1) - (1.0)) < 1e-12))
-            face->set_boundary_id(1);
-    }
+  for (auto &face : triangulation.active_face_iterators())
+      if (std::fabs(face->center()(1) - (-1.0)) < 1e-12 || 
+          std::fabs(face->center()(1) - (1.0)) < 1e-12)
+    face->set_boundary_id(1);
```

Comment:
Oh, my bad. I meant the blank space after `if`. Sorry!

Reply:
Ok.. I have added a new blank line after if. Please feel free to let me know if further changes are needed.